pages/common/peerflix.md

```diff
@@ -0,0 +1,15 @@
+# peerflix
+
+> Stream video or audio based torrents to your favorite player.
+
+- Stream all video-files in a torrent:
```

Comment:
This command does not have the `--all` flag. But still you have mentioned `all video-files`. If it will indeed stream all the files, then perhaps we do not need the last example at all ?

And also, it's better to just mention "files" instead of "video-files", as you don't know what a torrent will contain.